.circleci/config.yml

```diff
@@ -260,10 +262,10 @@ jobs:
       - persist_to_workspace:
           root: ~/wp-desktop
           paths: *app_cache_paths
-      - slack/status:
-          fail_only: true
-          mentions: '$pullRequestUserName'
-          failure_message: ':red_circle: wp-desktop tests for $CIRCLE_BRANCH have failed.\n\nCalypso PR: https://www.github.com/Automattic/wp-calypso/pull/$pullRequestNum'
+      # - slack/status:
```

Comment:
We know this works so just temporarily disabling for now while we test the PR review automation.